The-SMIRNOFF-force-field-format.md

```diff
@@ -31,6 +31,8 @@ Equilibrium angle values are provided, along with force constants (with units as
 
 This hierarchical structure means that a typical parameter file will tend to have generic parameters early in the section for each force type, with more specialized parameters assigned later.
 
+**QUESTION:** Can a SMIRNOFF section appear more than once? If so, what is the defined behavior there?
```

Comment:
As in, when separate force fields are loaded? Yes, can appear more than once in that case. Subsequent instances will be appended to the hierarchy, so loading, say, a water model at the very end with a SMIRKS pattern which only matches water will override general oxygen parameters which otherwise would be applied to hydrogen. The same will be true if someone wants to load specific parameters for ions which are tailored to work best with a particular water model. This is important since otherwise it'll be very hard for people to "swap" water models, ion models, etc., as they currently do. 

Likewise, someone who wishes to have very broad coverage of chemical space could load generic parameters that cover all bonds, angles, and torsions initially, then override with more specialized parameters which only apply to certain atom types.